packages/fxa-settings/src/components/FormPasswordWithBalloons/index.test.tsx

```diff
@@ -81,4 +87,20 @@ describe('FormPasswordWithBalloons component', () => {
       { timeout: SHOW_BALLOON_TIMEOUT + 200 }
     );
   });
+
+  // TODO in FXA-7482, review our password requirements and best way to display them
+  it('disallows space-only passwords', async () => {
+    renderWithLocalizationProvider(<Subject passwordFormType="signup" />);
+    const passwordField = screen.getByLabelText('Password');
+    fireEvent.focus(passwordField);
+    fireEvent.change(passwordField, { target: { value: '        ' } });
```

Comment:
I do believe https://testing-library.com/docs/user-event/intro/ is the new hotness for um, user events.